lib/TabTracker.js

```diff
@@ -121,6 +125,13 @@ TabTracker.prototype = {
     tab.on("deactivate", this.logDeactivate("unfocus"));
     tab.on("close", this.logClose("close"));
   },
+
+  onLoadComplete: function(subject, topic, data) {
```

Comment:
nit: `onPerfLoadComplete`. `TabTracker` also measures engagement stats and the function name could be confusing